http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellAppInstall.scala

```diff
@@ -139,9 +140,13 @@ class CromwellAppInstall[F[_]](config: CoaAppConfig,
       cbasDao.getStatus(baseUri, authHeader).handleError(_ => false)
     ).sequence.map(_.forall(identity))
 
-  private def toCromwellAppDatabaseNames(dbNames: List[String]): Option[CromwellAppDatabaseNames] =
-    (dbNames.find(_.startsWith("cromwell")), dbNames.find(_.startsWith("cbas")), dbNames.find(_.startsWith("tes")))
-      .mapN(CromwellAppDatabaseNames)
+  private def toCromwellAppDatabaseNames(
+    dbResources: List[WsmControlledDatabaseResource]
+  ): Option[CromwellAppDatabaseNames] =
+    (getAzureDatabaseName(dbResources, "cromwell"),
```

Comment:
could there be a problem with the `cromwellmetadata` db getting selected here?

Reply:
I don't think `cromwellmetadata` would appear here because the `dbResources` is fetched from WSM or created based on the databases that apps support and Cromwell app doesn't support `cromwellmetadata` database. It only support `cromwell`, `tes` and `cbas` database (see [here](https://github.com/DataBiosphere/leonardo/blob/9354890a356334202adff9d83074811f40ee328a/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellAppInstall.scala#L33-L38)).